docs/cep/windows/window-types/tumbling-expression.md

```diff
@@ -0,0 +1,31 @@
+---
+title: TUMBLING_EXPRESSION()
+---
+
+A batch (tumbling) length window that holds and process a number of events as specified in the `length`.
```

Comment:
```suggestion
A batch (tumbling) window that holds and process a number of events as specified in the `expression`.
```
The batch processing does not happen based on a fixed length but based on a expression here